js/modules/backup.js

```diff
@@ -492,17 +495,23 @@ async function writeAttachment(attachment, options) {
     }
   }
 
-  const encrypted = await crypto.encryptSymmetric(key, attachment.data);
+  if (!Attachment.hasData(attachment)) {
+    throw new TypeError('"attachment.data" is required');
+  }
+
+  const ciphertext = await crypto.encryptSymmetric(key, attachment.data);
 
   const writer = await createFileAndWriter(dir, filename);
   const stream = createOutputStream(writer);
-  stream.write(Buffer.from(encrypted));
+  stream.write(Buffer.from(ciphertext));
```

Comment:
Using `ciphertext` for consistency as there was naming overlap with `encrypted`. It’s also nicely symmetric with `plaintext`.